src/components/TranslateBoxes.js

```diff
@@ -2,33 +2,45 @@ import React from 'react';
 import Grid from '@material-ui/core/Grid';
 import Paper from '@material-ui/core/Paper';
 import TextField from '@material-ui/core/TextField';
+import Box from '@material-ui/core/Box';
 
 
 export default function TranslateBoxes({input, handleInputChange, output, classes}){
     return(
         <Paper className={classes.paper}>
-            <Grid container className={classes.basicGrid} spacing={1}>
+            <Grid container className={classes.basicGrid} spacing={0}>
                 <Grid item xs={6}>
-                    <TextField
-                    id="input-text-area"
-                    label="Input Code"
-                    multiline
-                    fullWidth
-                    value={input}
-                    onChange={handleInputChange}
-                    InputProps={{ disableUnderline: true }}
-                    />
+                    <Box className = {classes.box}>
+                        <TextField
+                        id="input-text-area"
+                        label="Input Code"
+                        multiline
+                        fullWidth
+                        value={input}
+                        onChange={handleInputChange}
+                        InputProps={{
+                            style: {fontFamily: "monospace"},
+                            disableUnderline: true
+                        }}
+                        />
+                    </Box>
                 </Grid>
                 <Grid item xs={6}>
-                    <TextField
-                    id="standard-multiline-static"
-                    label="Translation"
-                    multiline
-                    fullWidth
-                    InputProps={{ disableUnderline: true }}
-                    value={output}
-                    disabled
-                    />
+                    <Box className = {classes.box} bgcolor="grey.300">
```

Comment:
Consider making the text darker too to preserve contrast.

Reply:
Seems to be more complicated than I thought so I added a TODO